.github/workflows/cibuildwheel.yml

```diff
@@ -47,8 +47,9 @@ jobs:
           "include": [
             {"os": "ubuntu-latest", "CIBW_BUILD": "cp37-manylinux_*", "CIBW_ARCHS": "x86_64"},
             {"os": "ubuntu-latest", "CIBW_BUILD": "cp37-musllinux_*", "CIBW_ARCHS": "x86_64"},
-            {"os": "macos-12", "CIBW_BUILD": "cp37-*", "CIBW_ARCHS": "x86_64"},
-            {"os": "windows-2022", "CIBW_BUILD": "cp37-*", "CIBW_ARCHS": "AMD64"}
+            {"os": "macos-latest", "CIBW_BUILD": "cp37-*", "CIBW_ARCHS": "x86_64"},
+            {"os": "macos-latest", "CIBW_BUILD": "cp39-*", "CIBW_ARCHS": "arm64"},
```

Comment:
CI fails on wheel repair, specifically

```
delocate.delocating.DelocationError: Some missing architectures in wheel
  Required arch arm64 missing from /usr/local/Cellar/openssl@3/3.1.4/lib/libssl.3.dylib
  Required arch arm64 missing from libtorrent/__init__.cpython-39-darwin.so
```

so it looks like libtorrent wasn't built with arm64 support and openssl doesn't have it either. Here's how pygit2 went about it https://github.com/libgit2/pygit2/blob/master/.github/workflows/wheels.yml#L41C16-L41C16 , it builds openssl and itself before creating wheels `CIBW_BEFORE_ALL` and then uses a custom `CIBW_REPAIR_WHEEL_COMMAND_MACOS` to repair them. Looks we'll have to do the same here. Also might as well build universal wheels here instead of just `arm64`, so that we can drop pure `x86_64` later on.